src/codeview.jl

```diff
@@ -121,9 +122,10 @@ cthulhu_typed(io::IO, debuginfo::DebugInfo, args...; kwargs...) =
     cthulhu_typed(io, Symbol(debuginfo), args...; kwargs...)
 function cthulhu_typed(io::IO, debuginfo::Symbol,
     src::Union{CodeInfo,IRCode}, @nospecialize(rt), effects::Effects, mi::Union{Nothing,MethodInstance};
-    iswarn::Bool=false, hide_type_stable::Bool=false,
+    iswarn::Bool=false, hide_type_stable::Bool=false, optimize::Bool=true,
```

Comment:
Ooh, interesting, seems like this could be a separate PR (especially if you have a triggering issue that's independent of this PR). But I'm OK if it gets bundled here.

Reply:
I'm pretty sure the only reason I'm passing in `optimize` is because it's needed for the `find_callsites` function here, https://github.com/JuliaDebug/Cthulhu.jl/blob/c37e281901642989effad82d9e2f0b0d803fe487/src/codeview.jl#L325